corehq/apps/registration/utils.py

```diff
@@ -241,6 +205,31 @@ def send_new_request_update_email(user, requesting_ip, entity_name, entity_type=
         logging.warning("Can't send email, but the message was:\n%s" % message)
 
 
+def send_mobile_experience_reminder(recipient, full_name):
+    url = absolute_reverse("login")
+
+    params = {
+        "full_name": full_name,
+        "url": url,
+        'url_prefix': '' if settings.STATIC_CDN else 'http://' + get_site_domain(),
+    }
+    message_plaintext = render_to_string(
+        'registration/email/mobile_signup_reminder.txt', params)
+    message_html = render_to_string(
+        'registration/email/mobile_signup_reminder.html', params)
+
+    subject = ugettext('Visit CommCareHQ on your computer!')
+
+    try:
+        send_html_email_async.delay(subject, recipient, message_html,
+                                    text_content=message_plaintext,
+                                    email_from=settings.DEFAULT_FROM_EMAIL,
+                                    ga_track=True)
+    except Exception:
```

Comment:
I see we do this a couple other places, but don't think it's adding any value - I doubt anyone is combing the logs looking for this warning.

Reply:
I like the idea of having the log, but I agree that it's pretty smelly. I'll add a `raise` so it'll just bubble up anyway

Reply:
Does anything actually happen if an error is raised here? Doesn't look like the calling code has an error callback (which I think is the right choice: in the very unlikely case that this call fails, there's nothing the user can do about it, silently failing seems fine).

Reply:
Yeah, that's right. I didn't attach the sending of the email to any kind of result. I figured it would show up in sentry if something went wrong, but yeah, it's a silent fail